src/driver/dune_style.mli

```diff
@@ -0,0 +1 @@
+val of_dune_build : Fpath.t -> Packages.set
```

Comment:
The module name is enigmatic enough that I think this deserves some documentation.